Flow.Launcher/Languages/en.xaml

```diff
@@ -162,6 +162,8 @@
     <system:String x:Key="devtool">DevTools</system:String>
     <system:String x:Key="settingfolder">Setting Folder</system:String>
     <system:String x:Key="logfolder">Log Folder</system:String>
+    <system:String x:Key="clearlogfolder">Clear Logs</system:String>
+    <system:String x:Key="clearlogfolderMessage">Are you sure you want to delete the entire log?</system:String>
```

Comment:
"Are you sure you want to delete all logs" or "clear all logs" maybe better.

Reply:
"Are you sure you want to delete all logs" applied.